tensorflow/lite/micro/test_helpers.cc

```diff
@@ -243,6 +292,35 @@ const Model* BuildSimpleModelWithBranch() {
   return model_builder.BuildModel({t0}, {t3});
 }
 
+const Model* BuildModelWithOfflinePlanning(int number_of_tensors,
+    const int32_t* metadata_buffer,
+    std::vector<NodeConnection> node_conn) {
```

Comment:
We're aiming at making the tests run-able on target device. If std::vector is pulling in a large dependency it might not run in the current build system (we had -nostdlib) or it might be too large to fit in. 

Wondering if there is a way to avoid std::vector?

Reply:
I'll see if I can replace it with an array.